google/resource_google_project_iam_policy_test.go

```diff
@@ -353,6 +361,15 @@ func testAccCheckGoogleProjectIamPolicyExists(projectRes, policyRes, pid string)
 	}
 }
 
+func testAccCheckGoogleDefaultProjectIamPolicyIsMerged(projectRes, policyRes string) resource.TestCheckFunc {
```

Comment:
Instead of creating a new method, you can call the other method like this:

```go
config := testAccProvider.Meta().(*Config)
testAccCheckGoogleProjectIamPolicyIsMerged("google_project_iam_policy.acceptance", "data.google_iam_policy.admin", config.Project)
```